pypika/tests/test_query.py

```diff
@@ -120,6 +115,20 @@ def test_replace_orderby_table(self):
             'SELECT "customer" ' 'FROM "b" ' 'ORDER BY "customer"', str(query)
         )
 
+    def test_replace_tuple_table(self):
+        query = (
+            Query.from_(self.table_a)
+            .select(self.table_a.cost, self.table_a.revenue)
+            .where(Tuple(self.table_a.cost, self.table_a.revenue) == Tuple(1, 2))
+        )
+
+        query = query.replace_table(self.table_a, self.table_b)
+
+        self.assertEqual(
+            "SELECT " '"cost","revenue" ' 'FROM "b" ' 'WHERE ("cost","revenue")=(1,2)',
```

Comment:
Is this SQL string correct? It has multiple opening/closing single quotes.

Reply:
Fixed the formatting of the sql strings.